packages/env-tests/src/tests/attestation.ts

```diff
@@ -0,0 +1,137 @@
+import { StableTokenWrapper } from '@celo/contractkit/lib/wrappers/StableTokenWrapper'
+import { generateKeys, generateMnemonic } from '@celo/utils/lib/account'
+import { privateKeyToAddress } from '@celo/utils/lib/address'
+import { sleep } from '@celo/utils/lib/async'
+import { describe, test } from '@jest/globals'
+import BigNumber from 'bignumber.js'
+import Logger from 'bunyan'
+import twilio from 'twilio'
+import { Context } from '../context'
+import { envVar, fetchEnv } from '../env'
+import { getKey, ONE, TestAccounts } from '../scaffold'
+import {
+  getIdentifierAndPepper,
+  getPhoneNumber,
+  pollForMessagesAndCompleteAttestations,
+  reportErrors,
+  requestAttestationsFromIssuers,
+  requestMoreAttestations,
+} from '../shared/attestation'
+
+async function fundAttestationAccount(
+  context: Context,
```

Comment:
if we have context type `Context`, which imo should have a better name, we don't also need stableTokenWrapper as a parameter